### PR TITLE
[BLE] Add support to change BLE name from KMKKeyboard.go

### PR DESCRIPTION
**Pull request by dzervas**
The following can be used to change advertising BLE name:

```python
    keyboard.go(hid_type=HIDModes.BLE, ble_name='Lab68')
```

Also I'm passing keyword arguments from `KMKKeyboard.go` to `KMKKeyboard.hid_helper()` to allow for various possible configurations later on (USB pid/vid, bluetooth stuff, etc.)

### CODE REVIEW

**Comment by kdb424** [comment]
Looks good to me.

**Comment by kdb424** [comment]
If not given an name to advertise will this be an issue? I am waiting on my bluetooth boards to come in to test, but seeing as the only assignment that I see is in your keymap, I thought it was worth an ask.

**Comment by dzervas** [comment]
default name is `KMK`

**Comment by kdb424** [comment]
Totally missed that. Awaiting on final review by @klardotsh or a week from today, whatever comes first.

**Review by kdb424 — APPROVED** [review]
Looks reasonable to merge.

File: kmk/ble.py
Comment (klardotsh) [inline kmk/ble.py:27]: generally commented-out code doesn't stick around in the tree, but I'm not going to block merge over it. more an FYI for the future.

File: kmk/ble.py
Comment (klardotsh) [inline kmk/ble.py:30]: I'm not super familiar with the BLE API: why is `scan_response` now not needed?

**Review by klardotsh — COMMENTED** [review]
Is there a reason for the kwargs-tinkering throughout here? It doesn't look like it's ultimately used anywhere in this PR

**Comment by klardotsh** [comment]
I guess I forgot to mention in the review comment: this generally looks good, I'm only pausing final approval over the curiosity questions; this is explicitly *not* a blocked merge ultimately unless more questions come out of the above. Thanks for the code!

File: kmk/ble.py
Comment (dzervas) [reply kmk/ble.py:27]: Yea, it's a placeholder. TX power is not yet supported, but it should be in the near future

File: kmk/ble.py
Comment (dzervas) [reply kmk/ble.py:30]: it's already created by default and we don't need to keep track of more variables than we should. Memory is already very tight

**Comment by dzervas** [comment]
About the `**kwargs` the idea was to be add even more settings in the future, but now that i think of it, it's not need it.

Rebasing this MR is a bit of a nightmare RN (got to rebase 6+ branches) so if only this is left, merge it and I'll fix it on a later MR

**Comment by dzervas** [comment]
Oh wait, you mean the whole play where I feed down kwargs from `keyboard.go` to `BLEHID.__init__`. Yea, in order to pass some options during ONLY the initialization of HID class, that's what I came up with. A regular class attribute wouldn't do it, as the user could change it on runtime and expect it to be reflected on the controller

**Pull request merged**
